src/rhub/api/tower.py

```diff
@@ -113,10 +118,11 @@ def get_template(template_id):
 
     except Exception as e:
         logger.exception(e)
-        return problem(500, 'Server Error', 'Uknown server error, {e}')
+        return problem(500, 'Server Error', f'Uknown server error, {e}')
```

Comment:
Again, a super small thing which probably doesn't matter, but "Uknown" -> "Unknown"

Reply:
It does matter. If you notice a typo, report it. :+1: